File: HAMM.cpp
#include <iostream>
using namespace std;

int hamming (string dna1, string dna2)
{
  int count = 0;
  if (dna1.size()== dna2.size())
  {
    for (int i = 0; i<dna1.size(); i++)
    {
      if (dna1[i] != dna2[i])
      {
        count++;
      }
    }
    return count;
  }
}

void main ()
{
  string dna1, dna2;
  cin >> dna1;
  cin >> dna2;
  cout << "The hamming distance is: " << endl;
  cout << hamming(dna1, dna2) << endl;
}

File: HAMM.cs
using System;

public class Program
{
  public static long hamming(string DNA1, string DNA2)
  {
    long hamm = 0;
    for (int i = 0; i<DNA1.Length; i++)
    {
      if (DNA1[i] != DMA2[i])
      {
        hamm++;
      }
    }
    return hamm;
  }
  public static void Main()
  {
    string DNA1 = Console.ReadLine();
    string DNA2 = Console.ReadLine();
    Console.WriteLine(hamming(DNA1, DNA2));
  }
}
